### PR TITLE
[Doc] publish new loading overview docs

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+![Loading options overview](../../assets/loading_intro_overview.png)
+
+:::note
+
+In the following sections, "batch" or "batch loading" refers the loading of a large amount of data from a specified source all at a time into StarRocks, whereas "stream" or "streaming" refers to the continuous loading of data in real time.
+
+:::
+
+### Loading methods
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:32]:
```suggestion
## Loading methods
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+#### [Insert](../InsertInto.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:34]:
```suggestion
### [Insert](../InsertInto.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+**File format:**
+
+- INSERT INTO VALUES: SQL
+- INSERT INTO SELECT:
+  - INSERT INTO SELECT FROM `<table_name>`: StarRocks tables
+  - INSERT INTO SELECT FROM FILES(): Parquet and ORC
+
+**Data volume:** Not fixed (The data volume varies based on the memory size.)
+
+#### [Stream Load](../StreamLoad.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:58]:
```suggestion
### [Stream Load](../StreamLoad.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Batch load data from a local file system.
+
+**File format:** CSV and JSON
+
+**Data volume:** 10 GB or less
+
+#### [Broker Load](../../sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:66]:
```suggestion
### [Broker Load](../../sql-reference/sql-statements/data-manipulation/BROKER_LOAD.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+**Business scenario:**
+
+- Batch load data from HDFS or cloud storage like AWS S3, Microsoft Azure Storage, Google GCS, and S3-compatible storage (such as MinIO).
+- Batch load data from a local file system or NAS.
+
+**File format:** CSV, Parquet, ORC, and JSON (supported since v3.2.3)
+
+**Data volume:** Dozens of GB to hundreds of GB
+
+#### [Pipe](../../sql-reference/sql-statements/data-manipulation/CREATE_PIPE.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:77]:
```suggestion
### [Pipe](../../sql-reference/sql-statements/data-manipulation/CREATE_PIPE.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+
+This loading method is supported from v3.2 onwards.
+
+:::
+
+**File format:** Parquet and ORC
+
+**Data volume:** 100 GB to 1 TB or more
+
+#### [Routine Load](../../sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:91]:
```suggestion
### [Routine Load](../../sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+
+#### [Spark Load](../../sql-reference/sql-statements/data-manipulation/SPARK_LOAD.md)
+
+**Business scenario:** Batch load data of Apache Hive™ tables stored in HDFS by using Spark clusters.
+
+**File format:** CSV, Parquet (supported since v2.0), and ORC (supported since v2.0)
+
+**Data volume:** Dozens of GB to TBs
+
+### Ecosystem tools
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:107]:
```suggestion
## Ecosystem tools
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+
+#### [Kafka connector](../Kafka-connector-starrocks.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:109]:
```suggestion
### [Kafka connector](../Kafka-connector-starrocks.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Stream data from Kafka.
+
+#### [Spark connector](../Spark-connector-starrocks.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:113]:
```suggestion
### [Spark connector](../Spark-connector-starrocks.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Batch load data from Spark.
+
+#### [Flink connector](../Flink-connector-starrocks.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:117]:
```suggestion
### [Flink connector](../Flink-connector-starrocks.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 11]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Stream data from Flink.
+
+#### [SMT](../../integrations/loading_tools/SMT.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:121]:
```suggestion
### [SMT](../../integrations/loading_tools/SMT.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 12]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Load data from data sources such as MySQL, PostgreSQL, SQL Server, Oracle, Hive, ClickHouse, and TiDB through Flink.
+
+#### [DataX](../../integrations/loading_tools/DataX-starrocks-writer.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:125]:
```suggestion
### [DataX](../../integrations/loading_tools/DataX-starrocks-writer.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 13]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Synchronize data between various heterogeneous data sources, including relational databases (for example, MySQL and Oracle), HDFS, and Hive.
+
+#### [CloudCanal](../../integrations/loading_tools/CloudCanal.md)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:129]:
```suggestion
### [CloudCanal](../../integrations/loading_tools/CloudCanal.md)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 14]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Migrate or synchronize data from source databases (for example, MySQL, Oracle, and PostgreSQL) to StarRocks.
+
+#### [Kettle Connector](https://github.com/StarRocks/starrocks-connector-for-kettle)
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:133]:
```suggestion
### [Kettle Connector](https://github.com/StarRocks/starrocks-connector-for-kettle)
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 15]
```diff
@@ -0,0 +1,199 @@
...
+
+**Business scenario:** Integrate with Kettle. By combining Kettle's robust data processing and transformation capabilities with StarRocks's high-performance data storage and analytical abilities, more flexible and efficient data processing workflows can be achieved.
+
+### API
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:137]:
```suggestion
## API
```

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+
+#### Stream Load transaction interface
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/Loading_intro.md:139]:
```suggestion
### Stream Load transaction interface
```

**Review by DanRoscigno — DISMISSED** [review]
I think that Loading Methods should be at same level as Introduction

File: docs/en/loading/loading_introduction/loading_considerations.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+You can configure the following parameters in the configuration file **be.conf** of each BE:
+
+- `write_buffer_size`
+  
+  This parameter specifies the maximum memory block size. The default size is 100 MB. The loaded data is first written to a memory block on the BE. When the amount of data that is loaded reaches the maximum memory block size that you specify, the data is flushed to disk. You must specify a proper maximum memory block size based on your business scenario.
+
+  - If the maximum memory block size is exceedingly small, a large number of small files may be generated on the BE. In this case, query performance degrades. You can increase the maximum memory block size to reduce the number of files generated.
+  - If the maximum memory block size is exceedingly large, remote procedure calls (RPCs) may time out. In this case, you can adjust the value of this parameter based on your business needs.
+
+- `streaming_load_rpc_max_alive_time_sec`
```
Comment (DanRoscigno) [inline docs/en/loading/loading_introduction/loading_considerations.md:59]: This does not match the BE config page, is it 600 seconds or 1200? 

But: The real question is this: How do we maintain the setting on this page and the BE configuration page? Should we come up with a method for managing these parameters in one file and including them wherever needed?

For this PR let's just get the parameter corrected either here or on the BE page, and I will add this to the list of things to design a process around.

File: docs/en/loading/BrokerLoad.md [hunk 18]
```diff
@@ -390,7 +390,7 @@ The following table describes the parameters in `jobInfo`.
...
 | label         | The label of the load job.                                   |
-| state         | The status of the load job. Valid values:<ul><li>`PENDING`: The load job is in queue waiting to be scheduled.</li><li>`QUEUEING`: The load job is in the queue waiting to be scheduled.</li><li>`LOADING`: The load job is running.</li><li>`PREPARED`: The transaction has been committed.</li><li>`FINISHED`: The load job succeeded.</li><li>`CANCELLED`: The load job failed.</li></ul>For more information, see the "Asynchronous loading" section in [Overview of data loading](../loading/Loading_intro.md). |
+| state         | The status of the load job. Valid values:<ul><li>`PENDING`: The load job is in queue waiting to be scheduled.</li><li>`QUEUEING`: The load job is in the queue waiting to be scheduled.</li><li>`LOADING`: The load job is running.</li><li>`PREPARED`: The transaction has been committed.</li><li>`FINISHED`: The load job succeeded.</li><li>`CANCELLED`: The load job failed.</li></ul>For more information, see the "Asynchronous loading" section in [Loading considerations](./loading_introduction/loading_considerations.md). |
```
Comment (evelynzhaojie) [inline docs/en/loading/BrokerLoad.md:393]: see the "Asynchronous loading" section in [Loading considerations](./loading_introduction/loading_considerations.md).  ---  is this path correct? i think the .md file should be loading_concepts

File: docs/en/loading/Loading_data_template.md [hunk 19]
```diff
@@ -392,7 +392,7 @@ INSERT INTO user_behavior_declared
 
 ## More information
 
-- For more details on synchronous and asynchronous data loading please see the [overview of data loading](../loading/Loading_intro.md) documentation.
+- For more details on synchronous and asynchronous data loading, see [Loading considerations](./loading_introduction/loading_considerations.md) documentation.
```
Comment (evelynzhaojie) [inline docs/en/loading/Loading_data_template.md:395]: incorrect file, loading concepts, not loading considerations

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 20]
```diff
@@ -0,0 +1,199 @@
...
+- Ecosystem tools: StarRocks Connector for Apache Kafka® (Kafka connector for short), StarRocks Connector for Apache Spark™ (Spark connector for short), StarRocks Connector for Apache Flink® (Flink connector for short), and other tools such as SMT, DataX, CloudCanal, and Kettle Connector
+- API: Stream Load transaction interface
+
+These options each have its own advantages and support its own set of data source systems to pull from.
+
+This topic provides an overview of these options, along with comparisons between them to help you determine the loading option of your choice based on your data source, business scenario, data volume, data file format, and loading frequency.
+
+## Introduction to loading options
+
+This section mainly provides the characteristics and business scenarios of the loading options available in StarRocks.
```
Comment (evelynzhaojie) [inline docs/en/loading/loading_introduction/Loading_intro.md:22]: "provides" does not match "characteristics" --- describes

File: docs/en/loading/loading_introduction/Loading_intro.md [hunk 21]
```diff
@@ -0,0 +1,199 @@
...
+
+![Loading options overview](../../assets/loading_intro_overview.png)
+
+:::note
+
+In the following sections, "batch" or "batch loading" refers the loading of a large amount of data from a specified source all at a time into StarRocks, whereas "stream" or "streaming" refers to the continuous loading of data in real time.
```
Comment (evelynzhaojie) [inline docs/en/loading/loading_introduction/Loading_intro.md:28]: refers to

File: docs/zh/table_design/table_types/unique_key_table.md [hunk 22]
```diff
@@ -72,8 +72,9 @@ PROPERTIES (
...
-建表完成后，您可以创建多种导入作业，导入数据至表中。具体导入方式，请参见[导入概览](../../loading/Loading_intro.md)。
+建表完成后，您可以创建多种导入作业，导入数据至表中。具体导入方式，请参见[导入方案](../../loading/loading_introduction/Loading_intro.md)。
 
-> - 导入数据时，仅支持全部更新，即导入任务需要指明所有列，例如示例中的 `create_time`、`order_id`、`order_state` 和 `total_price` 四个列。
->
-> - 在设计导入频率时，建议以满足业务对实时性的要求为准。查询更新表的数据时，需要聚合多版本的数据，当版本过多时会导致查询性能降低。所以导入数据至更新表时，应该适当降低导入频率，从而提升查询性能。如果业务对实时性的要求是分钟级别，那么每分钟导入一次更新数据即可，不需要秒级导入。
+:::note
+
+- 导入数据时，仅支持全部更新，即导入任务需要指明所有列，例如示例中的 `create_time`、`order_id`、`order_state` 和 `total_price` 四个列。
+- 在设计导入频率时，建议以满足业务对实时性的要求为准。查询更新表的数据时，需要聚合多版本的数据，当版本过多时会导致查询性能降低。所以导入数据至更新表时，应该适当降低导入频率，从而提升查询性能。如果业务对实时性的要求是分钟级别，那么每分钟导入一次更新数据即可，不需要秒级导入。
```
Comment (evelynzhaojie) [inline docs/zh/table_design/table_types/unique_key_table.md:80]: lacks the closing ::: for note

File: docs/en/loading/loading_introduction/Loading_intro.md (hunk 1 above)
Comment (amber-create) [reply docs/en/loading/loading_introduction/Loading_intro.md:32]: No. Actually "loading options" consists of three parts, which are "loading methods", "ecosystem tools", and "API".

File: docs/en/loading/loading_introduction/Loading_intro.md (hunk 2 above)
Comment (amber-create) [reply docs/en/loading/loading_introduction/Loading_intro.md:34]: same reason as stated on line 32

File: docs/en/loading/loading_introduction/Loading_intro.md (hunk 4 above)
Comment (amber-create) [reply docs/en/loading/loading_introduction/Loading_intro.md:66]: same reason as stated on line 32

File: docs/en/loading/loading_introduction/Loading_intro.md (hunk 16 above)
Comment (evelynzhaojie) [inline docs/en/loading/loading_introduction/Loading_intro.md:139]: need a link to this doc

File: docs/en/loading/loading_introduction/loading_considerations.md (hunk 17 above)
Comment (amber-create) [reply docs/en/loading/loading_introduction/loading_considerations.md:59]: I've checked the latest BE parameter code repository, and you are right, the default parameter value now is 1200. Docs have been updated.

File: docs/en/loading/loading_introduction/Loading_intro.md (hunk 1 above)
Comment (DanRoscigno) [reply docs/en/loading/loading_introduction/Loading_intro.md:32]: No, loading options is four parts: intro, methods, tools, and API. Why would you consider methods to be a component of intro? Same question for tools and API?

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Pull request merged**